package.json

```
@@ -65,7 +65,7 @@
     "*.md"
   ],
   "engines": {
-    "node": "^18.0.0 || ^20.0.0"
+    "node": "^18.0.0 || ^20.0.0 || ^22.0.0-nightly20240313"
```

```suggestion
    "node": "^18.0.0 || ^20.0.0 || ^22.0.0"
```

use 22 when released